monkey/infection_monkey/config.py

```diff
@@ -232,6 +237,18 @@ def get_exploit_user_password_or_hash_product(self):
             cred_list.append(cred)
         return cred_list
 
+    @staticmethod
+    def hash_sensitive_data(sensitive_data):
+        """
+        Hash sensitive data (e.g. passwords). Used so the log won't contain sensitive data plain-text, as the log is
+        saved on client machines plain-text.
+
+        :param sensitive_data: the data to hash.
+        :return: the hashed data.
+        """
+        password_hashed = hashlib.sha512(sensitive_data.encode()).hexdigest()
+        return password_hashed
+
```

Comment:
Why is this here rather than in some common utils?

Reply:
I don't even know why we have a mix of data structure and methods on the same file. I just move the method next to parameters it modifies. This should be refactored by AT LEAST splitting config data structure from config service. But a bit out of scope.